<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From</class>
 <widget class="QWidget" name="From">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebChat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../web/static/images/xc.ico</normaloff>../web/static/images/xc.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size: 18px;
}
#From{
	padding: 0;
}
#BG{
	border-image: url(:/icon/static/images/bg.png) no-repeat;
	background-position: center center;
	background-size: cover;
}
#title{
	font-size: 28px;
	background-color: rgb(85, 170, 255);
}
#local_host{
	font-size: 23px;
	background-color: rgb(255, 255, 127);
}
#start_or_stop_btn{
	width: 70px;
	height: 30px;
	color: rgb(255, 255, 255);
	font-weight: bold;
	background-color: rgb(255, 0, 127);
}
#update_btn{
	background: url(:/icon/static/images/cx.ico) no-repeat;
}
#show_log{
    font-weight: bold;
    font-size: 17px;
	color: rgb(99, 99, 148);
	background-color: rgba(170, 170, 255, 0.5);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="BG">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="title">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>WebChat</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="local_host">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_or_stop_btn">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update_btn">
            <property name="maximumSize">
             <size>
              <width>37</width>
              <height>37</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="clear_all_btn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>ClearALL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="history_info">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>HistoryALL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="picture">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>picture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextBrowser" name="show_log">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../web/zy.qrc"/>
 </resources>
 <connections/>
</ui>
